### DISCUSSION TITLE
Can I use standalone runner from security-scan4x.zip to scan .NET 5+ project?

### DISCUSSION THREAD

**Discussion by JimQJak** [security-code-scan/security-code-scan#263] [Q&A]
> [Stand-alone runner](https://www.nuget.org/packages/security-scan/). Install with dotnet tool install --global security-scan and run security-scan /your/solution.sln. For older .NET 4.x please use security-scan4x.zip from [GitHub Releases](https://github.com/security-code-scan/security-code-scan/releases).

I'm just curious if I could use standalone runner from security-scan4x.zip to scan `>.NET 4.x` without need to install the scanner with `dotnet tool` command.

**Comment by JimQJak** [comment]
Hi @JarLob, really hope could you help answer this question

**Comment by JarLob** [comment]
Hi, I don't know. I suggest to try and see which one works better for you and which one throws errors. I have made the .net 4.x build because I noticed the dotnet tool was failing to analyze some of the older .net projects. However recently I have added a parameter to pass the sdk path. Maybe it can help.

**Comment by mwaddell** [comment]
Or, conversely - can the latest dotnet tool scan .net 4.x framework projects or will it miss things that the 4.x runner will catch?
